lib/hydra/derivatives/processors/active_encode.rb

```diff
@@ -0,0 +1,55 @@
+require 'active_encode'
+
+module Hydra::Derivatives::Processors
+  class ActiveEncodeError < StandardError
+    def initialize(status, source_path, errors = [])
+      msg = "ActiveEncode status was \"#{status}\" for #{source_path}"
+      msg = "#{msg}: #{errors.join(' ; ')}" unless errors.empty?
+      super(msg)
+    end
+  end
+
+  class ActiveEncode < Processor
+    class_attribute :timeout
+    attr_accessor :encode_class
```

Comment:
I think of encode more as a wrapper of an encoding job instead of the encoder itself so I've renamed accordingly.